nupic/research/frameworks/continuous_learning/dendrite_layers.py

```diff
@@ -150,52 +142,39 @@ def __init__(self,
         )
         self.output = DendriteOutput(out_dim, self.dendrites_per_neuron)
 
-        if self.act_fun_type == "kwinner":
-            self.act_fun = DendriteKWinners2d(
-                channels=self.out_dim,
-                k=1,
-                local=True,
-            )
-        else:
-            self.act_fun = torch.sigmoid
-
-    def forward(self, x, cat_projection=None):
+    def forward(self, x, cat_projection=1.0):
         """ cat_proj here is an optional argument
         for a categorical "feedback" projection to
         the dendrite segments
         """
-        if self.act_fun_type is None:
+        if self.act_fun_type == "kwinner":
             return self.forward_kwinner(x, cat_projection)
         else:
             return self.forward_sigmoid(x, cat_projection)
```

Comment:
Would recommend explicitly checking for "sigmoid" and then throwing an exception if it doesn't match "kwinner" or "sigmoid". This way a spelling mistake when calling it won't silently run as sigmoid.